components/servo/Cargo.toml

```diff
@@ -96,3 +96,10 @@ android_glue = "0.2"
 
 [target.'cfg(not(target_os = "windows"))'.dependencies]
 gaol = {git = "https://github.com/servo/gaol"}
+
+[build-dependencies]
+winres = "0.1"
+
+[package.metadata.winres]
+OriginalFilename = "SERVO.EXE"
+LeagalCopyright = "Copyright © 2016"
```

Comment:
Apparently this was a typo made by me in first version of the readme for winres. This hast to be `LegalCopyright`, or any other _string-name_ from https://msdn.microsoft.com/en-us/library/windows/desktop/aa381058.aspx

In the current version of https://github.com/mxre/winres this documentation typo has been fix a few weeks back